src/python3_pip_skeleton/__main__.py

```diff
@@ -106,6 +112,36 @@ def replace_in_file(file_path: Path, text_from: str, text_to: str):
             if child.suffix in CHANGE_SUFFIXES and child.name not in IGNORE_FILES:
                 text = replace_text(child.read_text())
                 child.write_text(text)
+            # Replace the file, ignoring line numbers
+            elif (
+                child.suffix in CHANGE_SUFFIXES
+                and child.name in IGNORE_FILES
+                and IGNORE_FILES[child.name]
+            ):
+                original_text = child.read_text()
+                replaced_text = replace_text(original_text)
+                for sub_strings in IGNORE_FILES[child.name]:
```

Comment:
Could: this probably works, but is changing everything, then changing specific sections back again, which feels a little fragile. When I did this previously I only changed the sections that weren't in the ignore range:
https://github.com/DiamondLightSource/python3-pip-skeleton-cli/blob/07569f30fd6a3a624cf28da434b1dfe2f84cca47/src/python3_pip_skeleton/__main__.py#L101-L121
You could do something similar here after you've done your finditer